MyLibrary/MyLibraryApp.DataAccess/DatabaseContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using MyLibraryApp.DataAccess.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.DataAccess
+{
+    public class DatabaseContext : DbContext
+    {
+        public DatabaseContext(DbContextOptions options) : base(options)
+        { }
+
+        public DbSet<Category> Categories { get; set; }
+        public DbSet<Book> Books { get; set; }
+        public DbSet<Order> Orders { get; set; }
+        public DbSet<OrderBook> OrderBooks { get; set; }
+        public DbSet<PublishingHouse> PublishingHouse { get; set; }
```

Comment:
The name should be `PublishingHouses`, so plural, in order to respect database naming best practices.